www/js/app.js

```diff
@@ -1561,6 +1582,29 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
 
     // A string to hold any anchor parameter in clicked ZIM URLs (as we must strip these to find the article in the ZIM)
     var anchorParameter;
+    
+    /**
+     * Warn the user that he/she clicked on an external link, and open it in a new tab
+     * 
+     * @param {Event} event the click event (on an anchor) to handle
+     */
+    function warnAndOpenExternalLinkInNewTab(event) {
+        event.preventDefault();
+        event.stopPropagation();
+        var target = event.target.target;
+        var message = 'Do you want to open this external link?';
+        if (!target || target === '_blank') {
+            message += ' (in a new tab)';
+        }
+        message += '<br />' + event.target.href + '<br />(this behavior can be disabled in settings if you know what you do)';
```

Comment:
Slight correction to English, the verb needs to use continuous tense:

`this behavior can be disabled in settings if you know what you are doing`

Or, because it's a bit wordy, it could be simplified to `this warning can be disabled in settings`.

Reply:
Btw, "behavior" is US spelling. In UK, it would be "behaviour". But use what you prefer, I'm not nationalistic!!
Or just use "warning".

Reply:
In fact, the setting not only affects the warning, but also the fact that it opens in a new tab.
And maybe we should not encourage that, as it can have bad side-effects.
So maybe I should simply remove this sentence